### PR TITLE
pkg(com.ape.mtbf): add Archive link for Twitter

### PR DESCRIPTION
**Pull request by TechFanTheo**

### CODE REVIEW

File: resources/assets/uad_lists.json [hunk 1]
```diff
@@ -31399,7 +31399,7 @@
...
+    "description": "Named 'MTBF Tools'\nDialer Code: 68238665. Uses SQLite Database. Reads/writes to External Storage. All sorts of permissions (camera, mic, battery, radio, led, sim, gps, nfc).\nFound more details at https://twitter.com/dbauduin/status/940126704261099520 (Archive : https://web.archive.org/web/20240330025312/https://twitter.com/dbauduin/status/940126704261099520)\nHardcoded URLs (an api and a book): http://www.andykhan.com/jexcelapi\nhttp://www.amazon.co.uk/exec/obidos/ASIN/0571058086/qid=1099836249/sr=1-3/ref=sr_1_11_3/202-6017285-1620664\nhttps://www.amazon.co.uk/exec/obidos/ASIN/0571058086qid=1099836249/sr=1-3/ref=sr_1_11_3/202-6017285-1620664",
```
Comment (AnonymousWP) [inline resources/assets/uad_lists.json:31402]:
```suggestion
    "description": "Named 'MTBF Tools'\nDialer Code: 68238665. Uses SQLite Database. Reads/writes to External Storage. All sorts of permissions (camera, mic, battery, radio, led, sim, gps, nfc).\nFound more details at https://twitter.com/dbauduin/status/940126704261099520 (archive: https://web.archive.org/web/20240330025312/https://twitter.com/dbauduin/status/940126704261099520)\nHardcoded URLs (an api and a book): http://www.andykhan.com/jexcelapi\nhttp://www.amazon.co.uk/exec/obidos/ASIN/0571058086/qid=1099836249/sr=1-3/ref=sr_1_11_3/202-6017285-1620664\nhttps://www.amazon.co.uk/exec/obidos/ASIN/0571058086qid=1099836249/sr=1-3/ref=sr_1_11_3/202-6017285-1620664",
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

**Comment by TechFanTheo** [comment]
Sorry I didn't now I had to approve the suggestion. Look like it is outdated and cannot be aplied.

**Comment by AnonymousWP** [comment]
> Sorry I didn't now I had to approve the suggestion. Look like it is outdated and cannot be aplied.

In that case you need to push a new commit from your local system to this PR. Just change it on your computer and then push.

**Comment by AnonymousWP** [comment]
@TechFanTheo See above.

**Pull request closed without merging**